inc/namespace.php

```diff
@@ -193,6 +193,19 @@ function get_environment_type() : string {
 	return 'local';
 }
 
+/**
+ * Get the region of the current environment.
```

Comment:
Thinking about it we could make this a bit clearer that it's the AWS region:
```suggestion
 * Get the AWS region of the current environment.
```

Also add some examples like in the doc block for the function above eg. `us-east-1`, `ap-northeast-1`, `eu-central-1` etc